lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -123,3 +128,29 @@ function dosomething_rogue_get_by_file_id($fid)
 {
   return db_query("SELECT rogue_rbs.rogue_item_id FROM {dosomething_rogue_reportbacks} rogue_rbs WHERE fid = :fid", array(':fid' => $fid))->fetchAll();
 }
+
+/**
+ * Insert record that stores reference to the most recent uploaded reportback item in
+ * phoenix and it's corresponding id's in Rogue
+ *
+ * @param string $rbid
+ * @param string $fid
+ * @param object $rogue_reportback
+ *
+ * @return InsertQuery object
```

Comment:
I don't think we have any other instances of an insert living alone in a function. 😂